installation from github : File format not recognized

When I install from github.  nm.exe is  printing the message
```
File format not recognized
```

```r
R version 3.5.1 Patched (2018-10-09 r75424) -- "Feather Spray"
Copyright (C) 2018 The R Foundation for Statistical Computing
Platform: x86_64-w64-mingw32/x64 (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

  Natural language support but running in an English locale

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.

> devtools::install_github("r-lib/rlang", force = TRUE)
Downloading GitHub repo r-lib/rlang@master
from URL https://api.github.com/repos/r-lib/rlang/zipball/master
Installing rlang
"W:/R-3.5._/App/R-Portable/bin/x64/R" --no-site-file --no-environ --no-save  \
  --no-restore --quiet CMD INSTALL  \
  "W:/R-3.5._/R_USER_3.5.__R_STUDIO/AppData/Local/Temp/RtmpKwA6E6/devtools3ab067e0391d/r-lib-rlang-30d6671"  \
  --library="W:/R-3.5._/R_LIBS_USER_3.5._" --with-keep.source --install-tests

In R CMD INSTALL

* installing *source* package 'rlang' ...
** libs

*** arch - i386
W:/Rtools35/mingw_32/bin/gcc  -I"W:/R-35~1._/App/R-PORT~1/include" -DNDEBUG -I./lib/         -O3 -Wall  -std=gnu99 -mtune=generic -c capture.c -o capture.o
W:/Rtools35/mingw_32/bin/gcc  -I"W:/R-35~1._/App/R-PORT~1/include" -DNDEBUG -I./lib/         -O3 -Wall  -std=gnu99 -mtune=generic -c export.c -o export.o
W:/Rtools35/mingw_32/bin/gcc  -I"W:/R-35~1._/App/R-PORT~1/include" -DNDEBUG -I./lib/         -O3 -Wall  -std=gnu99 -mtune=generic -c internal.c -o internal.o
W:/Rtools35/mingw_32/bin/gcc  -I"W:/R-35~1._/App/R-PORT~1/include" -DNDEBUG -I./lib/         -O3 -Wall  -std=gnu99 -mtune=generic -c lib.c -o lib.o
W:\Rtools35\mingw_32\bin\nm.exe: lib.c: File format not recognized
W:\Rtools35\mingw_32\bin\nm.exe: internal.c: File format not recognized
W:\Rtools35\mingw_32\bin\nm.exe: export.c: File format not recognized
W:/Rtools35/mingw_32/bin/gcc -shared -s -static-libgcc -o rlang.dll tmp.def capture.o export.o internal.o lib.o -LW:/R-35~1._/App/R-PORT~1/bin/i386 -lR
installing to W:/R-3.5._/R_LIBS_USER_3.5._/rlang/libs/i386

*** arch - x64
W:/Rtools35/mingw_64/bin/gcc  -I"W:/R-35~1._/App/R-PORT~1/include" -DNDEBUG -I./lib/         -O2 -Wall  -std=gnu99 -mtune=generic -c capture.c -o capture.o
W:/Rtools35/mingw_64/bin/gcc  -I"W:/R-35~1._/App/R-PORT~1/include" -DNDEBUG -I./lib/         -O2 -Wall  -std=gnu99 -mtune=generic -c export.c -o export.o
W:/Rtools35/mingw_64/bin/gcc  -I"W:/R-35~1._/App/R-PORT~1/include" -DNDEBUG -I./lib/         -O2 -Wall  -std=gnu99 -mtune=generic -c internal.c -o internal.o
In file included from ./lib/rlang.h:75:0,
                 from internal/arg.c:1,
                 from internal.c:1:
internal/eval-tidy.c: In function 'rlang_tilde_eval':
./lib/env.h:36:3: warning: 'top' may be used uninitialized in this function [-Wmaybe-uninitialized]
   SET_ENCLOS(env, new_parent);
   ^
In file included from internal.c:5:0:
internal/eval-tidy.c:372:9: note: 'top' was declared here
   sexp* top;
         ^
W:/Rtools35/mingw_64/bin/gcc  -I"W:/R-35~1._/App/R-PORT~1/include" -DNDEBUG -I./lib/         -O2 -Wall  -std=gnu99 -mtune=generic -c lib.c -o lib.o
W:\Rtools35\mingw_64\bin\nm.exe: lib.c: File format not recognized
W:\Rtools35\mingw_64\bin\nm.exe: internal.c: File format not recognized
W:\Rtools35\mingw_64\bin\nm.exe: export.c: File format not recognized
. . .
```

--- a/src/Makevars
+++ b/src/Makevars
@@ -47,7 +47,7 @@ export-files = \
 
 all: $(SHLIB)
 
-$(SHLIB): lib.c internal.c export.c
+$(SHLIB): lib.o internal.o export.o
 
 lib.c: $(lib-files)
 	touch lib.c

